portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -603,6 +603,94 @@ definition {
 			resourcePanels = "Portal Services,Sample QA app");
 	}
 
+	@description = "This is a use case for LRQA-57134."
+	test RefreshTokenTime {
+		property app.server.types = "jboss,tcserver,tomcat,weblogic,websphere,wildfly";
+		property database.types = "db2,hypersonic,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
+		property custom.properties = "virtual.hosts.default.site.name=Guest";
+
+		var clientId = "abc123";
+		var clientSecret = "abc123";
+		var portalURL = PropsUtil.get("portal.url");
+
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "System Settings");
+
+		SystemSettings.gotoConfiguration(
+			configurationCategory = "OAuth 2",
+			configurationName = "Default BearerTokenProvider",
+			configurationScope = "System Scope");
+
+		SystemSettings.editTextSetting(
+			settingName = "Access Token Duration",
+			settingValue = "5");
+
+		SystemSettings.saveConfiguration();
```

Comment:
Let's use `osgi.module.configuration.file.names` and `osgi.module.configurations` to configure this instead of going to System Setting. This makes the test less susceptible to flakiness.